dask/dataframe/io/io.py

```diff
@@ -570,10 +570,11 @@ def from_delayed(
 
     Parameters
     ----------
-    dfs : list of Delayed
+    dfs : list of Delayed or Future
         An iterable of ``dask.delayed.Delayed`` objects, such as come from
-        ``dask.delayed`` These comprise the individual partitions of the
-        resulting dataframe.
+        ``dask.delayed`` or an iterable of ``concurrent.futures.Future`` objects,
+        such as come from ``concurrent.futures``. These comprise the individual
```

Comment:
And, `Future` objects are created by [all of these functions](https://distributed.dask.org/en/stable/memory.html?highlight=future%20object#creating-futures), so based on [the language used here](https://distributed.dask.org/en/stable/#architecture), maybe we can say:

"such as come from `client.submit` interface."

Reply:
Yes, that seems correct. I will make the change to reflect it.